tests/manifest.yaml

```diff
@@ -1,140 +1,225 @@
-  stdoutFile: fixtures/espresso/test-001/pw-scf.out
-
-test_espresso_total_energy_contributions:
-  workDir: fixtures/espresso/test-001
-  stdoutFile: fixtures/espresso/test-001/pw-scf.out
-
-test_espresso_phonon_dos:
-  workDir: fixtures/espresso/test-003
-  stdoutFile: fixtures/espresso/test-003/phonon_dos.out
-
-test_espresso_phonon_dispersion:
-  workDir: fixtures/espresso/test-003
-  stdoutFile: fixtures/espresso/test-003/normal_modes.out
-
-test_vasp_total_energy:
-  workDir: fixtures/vasp/test-001
-  stdoutFile: fixtures/vasp/test-001/vasp.out
-
-test_vasp_fermi_energy:
-  workDir: fixtures/vasp/test-001
-  stdoutFile: fixtures/vasp/test-001/vasp.out
-
-test_vasp_nspins:
-  workDir: fixtures/vasp/test-001
-  stdoutFile: fixtures/vasp/test-001/vasp.out
-
-test_vasp_eigenvalues_at_kpoints:
-  workDir: fixtures/vasp/test-001
-  stdoutFile: fixtures/vasp/test-001/vasp.out
-
-test_vasp_ibz_k_points:
-  workDir: fixtures/vasp/test-001
-  stdoutFile: fixtures/vasp/test-001/vasp.out
-
-test_vasp_dos:
-  workDir: fixtures/vasp/test-002
-  stdoutFile: fixtures/vasp/test-001/chgcar.out
-
-test_vasp_convergence_electronic:
-  workDir: fixtures/vasp/test-001
-  stdoutFile: fixtures/vasp/test-001/vasp.out
-
-test_vasp_convergence_ionic:
-  workDir: fixtures/vasp/test-001
-  stdoutFile: fixtures/vasp/test-001/vasp.out
-
-test_vasp_stress_tensor:
-  workDir: fixtures/vasp/test-001
-  stdoutFile: fixtures/vasp/test-001/vasp.out
-
-test_vasp_pressure:
-  workDir: fixtures/vasp/test-001
-  stdoutFile: fixtures/vasp/test-001/vasp.out
-
-test_vasp_total_force:
-  workDir: fixtures/vasp/test-001
-  stdoutFile: fixtures/vasp/test-001/vasp.out
-
-test_vasp_atomic_forces:
-  workDir: fixtures/vasp/test-001
-  stdoutFile: fixtures/vasp/test-001/vasp.out
+tests:
+  # Main definition for tests. An application can override these settings if defined in the `applications` yaml
+  total_energy:
+    comparison: assertAlmostEqual
+  fermi_energy:
+    comparison: assertAlmostEqual
+  nspins:
+    comparison: assertDeepAlmostEqual
+  eigenvalues_at_kpoints:
+    comparison: assertDeepAlmostEqual
+  ibz_k_points:
+    comparison: assertDeepAlmostEqual
+  convergence_electronic:
+    comparison: assertDeepAlmostEqual
+  convergence_ionic:
+    comparison: assertDeepAlmostEqual
+  stress_tensor:
+    comparison: assertDeepAlmostEqual
+  pressure:
+    comparison: assertAlmostEqual
+  total_force:
+    comparison: assertAlmostEqual
+  atomic_forces:
+    comparison: assertDeepAlmostEqual
+  total_energy_contributions:
+    comparison: assertDeepAlmostEqual
+  dos:
+    comparison: assertDeepAlmostEqual
+  phonon_dos:
+    comparison: assertDeepAlmostEqual
+  phonon_dispersions:
+    comparison: assertDeepAlmostEqual
+  magnetic_moments:
+    comparison: assertDeepAlmostEqual
+  reciprocal_lattice:
+    comparison: assertDeepAlmostEqual
+  realspace_lattice:
+    comparison: assertDeepAlmostEqual
+
+test_config_schema:
+  name: name of property to test
```

Comment:
in schema we often describe the type too - int, string... Maybe:
```
- name:
   description: name of property to test
   type: String
```

Reply:
Updated the test_config schema in the manifest.yaml